geode-assembly/geode-assembly-test/src/main/java/org/apache/geode/session/tests/ServerContainer.java

```diff
@@ -128,7 +128,7 @@ public ServerContainer(ContainerInstall install, File containerConfigHome,
     configuration.setProperty(GeneralPropertySet.LOGGING, loggingLevel);
     // Removes secureRandom generation so that container startup is much faster
     configuration.setProperty(GeneralPropertySet.JVMARGS,
-        "-Djava.security.egd=file:/dev/./urandom -Xmx128m -Xms64m");
+        "-Djava.security.egd=file:/dev/./urandom -Xmx256m -Xms128m");
```

Comment:
Looks good and seems to pass consistently!  I think we can leave the `-Xms` at 64 though.